client/signup/jetpack-connect/sso.jsx

```diff
@@ -41,8 +61,22 @@ const JetpackSSOForm = React.createClass( {
 		this.props.errorNotice( 'Jetpack SSO is currently in development.' );
 	},
 
+	isButtonDisabled() {
+		const { nonceValid, isAuthorizing, isValidating, ssoUrl } = this.props;
+		return ! nonceValid || !! isAuthorizing || !! isValidating || !! ssoUrl;
```

Comment:
we could write it like this too :) `!! ( ! nonceValid || isAuthorizing || isValidating || ssoUrl )`